terraform/install.sh

```diff
@@ -214,7 +214,14 @@ installMonitoring() {
 
   gcp_monitoring_path="https://console.cloud.google.com/monitoring?project=$project_id"
   if [[ -z $skip_workspace_prompt ]]; then
+    log ""
+    log ""
+    log "********************************************************************************"
+    log ""
+    log "ATTENTION! ACTION REQUIRED!"
```

Comment:
I'd probably prefer an emoji instead of this: ⚠️

Or a mouse, to indicate action is required: 🖱️

Maybe in-line with the log text?

Reply:
Added emoji instead!